src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
 public class WorkWithFile {
```

Comment:
The access modifier for the 'getStatistic' method should not be public if it's only intended to be used within the class or package. Consider changing it to 'private' or package-private if that's the case.